supabase/migrations/20240725185224_crescendo-schema.sql

```diff
@@ -47,9 +47,13 @@ create table public.stand_form (
     scouter                 uuid not null references public.profile(id),
     auto_amp_notes          smallint not null default 0,
     auto_speaker_notes      smallint not null default 0,
+    auto_notes_missed     smallint not null default 0,
+    auto_shuttle_notes           smallint not null default 0,
     teleop_amp_notes        smallint not null default 0,
     teleop_speaker_notes    smallint not null default 0,
-    shuttle_notes           smallint not null default 0,
+    teleop_amped_speaker_notes smallint not null default 0,
```

Comment:
I've thought about it a bit, and I'm not sure what value we get from collection notes missed / amped speaker notes. We never really used data derived from notes missed, and I think all we got from amped notes was a bit better score prediction.

I think we'd get more accurate data by removing these fields